indico/modules/rb_new/client/js/components/Room.jsx

```diff
@@ -39,68 +40,30 @@ export default class Room extends React.Component {
         isFavorite: false
     };
 
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            dimmed: false
-        };
-    }
-
-    shouldComponentUpdate(nextProps, nextState) {
+    shouldComponentUpdate(nextProps) {
         const {isFavorite: nextIsFavorite, room: nextRoom} = nextProps;
         const {room, isFavorite} = this.props;
-        const {dimmed} = this.state;
-        const {dimmed: nextDimmed} = nextState;
 
-        return nextIsFavorite !== isFavorite || !_.isEqual(room, nextRoom) || dimmed !== nextDimmed;
+        return nextIsFavorite !== isFavorite || !_.isEqual(room, nextRoom);
```

Comment:
I am sure this will cause "wasted renders". Maybe it would be better to keep it somehow?

Reply:
I removed the variable that we don't use anymore, did you think I removed the whole line?